tutorials/shaders/shader_reference/sky_shader.rst

```diff
@@ -154,25 +154,25 @@ There are 4 ``LIGHTX`` lights, accessed as ``LIGHT0``, ``LIGHT1``, ``LIGHT2``, a
 |                                 | frame.                                                                                                                   |                          
 +---------------------------------+--------------------------------------------------------------------------------------------------------------------------+
-| in vec3 **POSITION**            | Camera position in world space                                                                                           |
+| in vec3 **POSITION**            | Camera position, in world space.                                                                                         |
 +---------------------------------+--------------------------------------------------------------------------------------------------------------------------+
 | samplerCube **RADIANCE**        | Radiance cubemap. Can only be read from during background pass. Check ``!AT_CUBEMAP_PASS`` before using.                 |
 +---------------------------------+--------------------------------------------------------------------------------------------------------------------------+
-| in bool **AT_HALF_RES_PASS**    | Currently rendering to half resolution pass.                                                                             |
+| in bool **AT_HALF_RES_PASS**    | ``true`` when rendering to half resolution pass.                                                                         |
 +---------------------------------+--------------------------------------------------------------------------------------------------------------------------+
-| in bool **AT_QUARTER_RES_PASS** | Currently rendering to quarter resolution pass.                                                                          |
+| in bool **AT_QUARTER_RES_PASS** | ``true`` when rendering to quarter resolution pass.                                                                      |
 +---------------------------------+--------------------------------------------------------------------------------------------------------------------------+
-| in bool **AT_CUBEMAP_PASS**     | Currently rendering to radiance cubemap.                                                                                 |
+| in bool **AT_CUBEMAP_PASS**     | ``true`` when rendering to radiance cubemap.                                                                             |
 +---------------------------------+--------------------------------------------------------------------------------------------------------------------------+
-| in bool **LIGHTX_ENABLED**      | ``LightX`` is visible and in the scene. If ``false``, other light properties may be garbage.                             |
+| in bool **LIGHTX_ENABLED**      | ``true`` if ``LIGHTX`` is visible and in the scene. If ``false``, other light properties may be garbage.                 |
```

Comment:
Maybe would be better calling these "LIGHT#"

Reply:
It's a little more readable in the docs, sure.

A side effect is that if you paste "LIGHT#_ENABLED" directly, you get an `Invalid symbols placed before directive` error. If you paste "LIGHTX_ENABLED" directly, you get an `Unknown identifier in expression: 'LIGHTX_COLOR'` error. So the error is more readable if we leave this as is, and will catch some nonzero amount of copy paste errors.

Reply:
I don't have any preference here. So I will leave it up to tetrapod00

Reply:
Leaving these as-is, erring on the side of fewer changes, and because of the the minor improvement in error clarity.